.github/workflows/ci.yml

```diff
@@ -77,6 +77,7 @@ jobs:
         if: github.repository_owner == 'getsentry'
         run: |
           docker build . \
+            --build-arg BUILDKIT_INLINE_CACHE=1 \
```

Comment:
This argument is the one that fixes the Docker caching not working from properly. See [docs](https://docs.docker.com/engine/reference/commandline/build/#specifying-external-cache-sources).

We removed `BUILDKIT_INLINE_CACHE=1` back in [here](https://github.com/getsentry/snuba/pull/2017/files#r674222449). If an image was not built with that argument then we don't have metadata that would allow it to be used with `--cache-from`, thus, when I fixed #2175 we started having slow Docker builds.